sdx_controller/swagger/swagger.yaml

```diff
@@ -174,8 +174,8 @@ paths:
         schema:
           maximum: 10
           minimum: 1
-          type: integer
-          format: int64
+          type: string
+          format: uuid
```

Comment:
if we use string type uuid here, I think we need to remove the min and max.

Reply:
Ah thanks! I assumed that `minimum` and `maximum` are constraints on the number of parameters, not their value itself.

Reply:
Removed `minimum` and `maximum`, and dropped `format: uuid`.  Connection IDs can be simple strings.  Also it looks like connexion (the version of connexion we use) does not care about `format` anyway.